internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -214,7 +226,10 @@ func (handler StorageHandler) newCommandValue(resourceName string, reading inter
 		if !ok {
 			return nil, fmt.Errorf(castError, resourceName, "not []byte")
 		}
-		result, errn = models.NewCommandValue(resourceName, timestamp, val, valueType)
+		Base64Val := convertToBase64(val)
```

Comment:
How about converting only image/jpeg and image/png `media types` to base64 rather than changing all binary data?

Reply:
Sure, I think that can be a better approach. I will add a check for that.

Reply:
Done.